app/views/questions/file_with_help/edit.html.erb

```diff
@@ -0,0 +1,40 @@
+<% content_for :page_title, "File with the help of a tax expert!" %>
+
+<% content_for :form_card do %>
+  <div class="form-card">
+    <h1 class="form-question">
+      <%= content_for(:page_title) %>
+    </h1>
+
+    <p>
+      You selected our full service option. We’ll have you answer a series of questions about your tax situation (to the best of your ability) and have a tax specialist review everything with you.
+    </p>
+
+    <p>
+    <strong>This is the right option for you if:</strong>
+    </p>
+
+    <ul class="checkmark-list">
+      <li>
```

Comment:
This could be made into some sort of partial or helper but in the end I did not think that much would be gained.

Reply:
we may be able to use other examples of a list of items with an icon for each

Reply:
@bengolder Are you suggesting this as a future refactoring when we need it or something to prepare for now?

Reply:
@jonsgreen non-blocking